#
#  Copyright (c) 2022 Composiv.ai, Eteration A.S. and others
#
# All rights reserved. This program and the accompanying materials
# are made available under the terms of the Eclipse Public License v2.0
# and Eclipse Distribution License v1.0 which accompany this distribution.
#
# The Eclipse Public License is available at
#    http://www.eclipse.org/legal/epl-v10.html
# and the Eclipse Distribution License is available at
#   http://www.eclipse.org/org/documents/edl-v10.php.
#
# Contributors:
#    Composiv.ai, Eteration A.S. - initial API and implementation
#
#
/**:
  ros__parameters:
    name: nacid-docker-01
    namespace: org.eclipse.muto.sandbox # subject to change
    stack_topic: /stack
    twin_topic: /twin
    type: simulator
    twin_url: "http://ditto:ditto@sandbox.composiv.ai"
    anonymous: False  # Use this for automatically generated id (uuid based)
  #   if anonymous is True or anynoymous param is missing, name/id will be auto generated